spacy_llm/ty.py

```diff
@@ -55,24 +58,69 @@ def from_disk(
         ...
 
 
+class FewshotExample(abc.ABC, BaseModel):
+    @classmethod
+    @abc.abstractmethod
+    def generate(cls, example: Example, **kwargs) -> Self:
+        """Create a fewshot example from a spaCy example.
+        example (Example): spaCy example.
+        """
+
+
 @runtime_checkable
-class Scorable(Protocol):
+class ScorableTask(Protocol):
+    """Differs from ScorableCallableProtocol in that it describes an object with a scorer() function, i. e. a scorable
```

Comment:
What is `ScorableCallableProtocol`?

Reply:
Fixed in https://github.com/explosion/spacy-llm/pull/242/commits/b687d54a5b3f72dfd05f848e490c140b45d70a3b.